site/content/en/docs/Installation/Terraform/gke.md

```diff
@@ -118,12 +121,32 @@ Next, make sure that you can authenticate using gcloud:
 ```
 gcloud auth application-default login
 ```
-
+{{% feature expiryVersion="1.9.0" %}}
 Now you can create your GKE cluster (optionally specifying the version of Agones you want to use):
 ```
-terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.0.0"]
+terraform apply -var project="<YOUR_GCP_ProjectID>" [-var agones_version="1.8.0"]
```

Comment:
I'm not sure why we are talking about agones_version here? it's the helm module that installs Agones. Where is this variable coming from?

(Also, release branches should line up with versions, so this should be handled with care, in case people mismatch things)

Reply:
or to be clearer - unless there is good reasons to put ` [-var agones_version="1.8.0"]` here, I don't think it makes sense, and should be removed.

:facepalm: 
oooh, I see - we had this here before, that's the context .I see that now.

Yeah, I'd recommend removing it to be honest. It's not contextual to the task at hand, and I would argue, likely confusing, since there is nothing in the GKE module that lets you install Agones.